### PR TITLE
✨ Implement sticky ad bottom type ad on amp-ad

### PR DESCRIPTION
**Pull request by powerivq**
Add a white background and shadow css, so that the amp-ad sticky ad looks much more like amp-sticky-ad. Ensure resizing works on both amp-ad scenarios.

### CODE REVIEW

File: extensions/amp-ad/0.1/amp-ad-ui.js [hunk 1]
```diff
@@ -315,14 +345,36 @@ export class AmpAdUIHandler {
...
+          divertStickyAdTransition(this.baseInstance_.win);
+          if (
+            getExperimentBranch(
+              this.baseInstance_.win,
+              STICKY_AD_TRANSITION_EXP.id
+            ) === STICKY_AD_TRANSITION_EXP.experiment
+          ) {
+            this.setSize_(this.element_.querySelector('iframe'), height, width);
+          }
+          return resizeInfo;
```
Comment (calebcordry) [inline extensions/amp-ad/0.1/amp-ad-ui.js:358]: unused return?

File: extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.css [hunk 2]
```diff
@@ -43,21 +43,23 @@ amp-story-page[xdomain-ad] .i-amphtml-glass-pane {
 }
 
 /* TODO(ccordry): refactor centering logic in amp-ad.css and remove this hack.  */
```
Comment (calebcordry) [inline extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.css:45]: lol.

File: extensions/amp-ad/0.1/amp-ad.css [hunk 3]
```diff
@@ -52,6 +52,12 @@ amp-embed iframe {
 
 amp-ad[sticky] {
   visibility: hidden;
+  align-items: center;
+}
+
+amp-ad[amp-story] iframe {
```
Comment (calebcordry) [inline extensions/amp-ad/0.1/amp-ad.css:58]: this should probably be in story ad css?

File: extensions/amp-ad/0.1/amp-ad-ui.js [hunk 4]
```diff
@@ -315,14 +345,36 @@ export class AmpAdUIHandler {
...
   }
 
+  /**
+   * Force set the dimensions for an element
+   * @param {Any} element
+   * @param {number} newHeight
+   * @param {number} newWidth
+   */
+  setSize_(element, newHeight, newWidth) {
+    setStyle(element, 'height', newHeight, 'px');
```
Comment (calebcordry) [inline extensions/amp-ad/0.1/amp-ad-ui.js:374]: nit: setStyles?

**Review by calebcordry — APPROVED** [review]

File: extensions/amp-ad/0.1/amp-ad-ui.js (hunk 1 above)
Comment (calebcordry) [reply extensions/amp-ad/0.1/amp-ad-ui.js:358]: yep, I was looking at the following lines as a `.then` but its actually the catch, oops.

File: extensions/amp-ad/0.1/amp-ad.css (hunk 3 above)
Comment (powerivq) [reply extensions/amp-ad/0.1/amp-ad.css]: moved.

**Comment by powerivq** [comment]
Merging was blocked by https://github.com/ampproject/amphtml/issues/31756

**Review by jridgewell — APPROVED** [review]

**Pull request merged**